### PR TITLE
Integrate news parsing endpoint

### PR DESCRIPTION
**Pull request by cesarvarela**
Deals with part of #104

### CODE REVIEW

**Comment by smcgregor** [comment]
* Good UI and progress, but needs more work before this checkpoint will be fully functional and mergeable.
*  This needs error handling before this checkpoint will be mergeable. A cold request to the API will often generate a 500 error (as it did for me)
* It looks like the maintext is not being placed into the form properly.
* Don't make the button disappear after fetching.
* Any affordance that lets the user know that the first request can take a while would be helpful. Subsequent requests are much faster.

**Comment by cesarvarela** [comment]
Regarding the cold start, there are 2 toasts:

- if the function returns 500, it says: `Error fetching news info, please try again in a few seconds.` (there is no way to tell if the function crashed or it was a cold start)
- if the function takes > 20 seconds, a warning toast that says: `Sometimes fetching news info may take a while...`

Not sure if you have in mind something more involved than that?

**Comment by smcgregor** [comment]
That should work.

File: site/gatsby-site/src/components/forms/IncidentReportForm.js [hunk 1]
```diff
@@ -110,8 +110,74 @@ const IncidentReportForm = ({ incident, onUpdate, onSubmit }) => {
...
+      severity: SEVERITY.warning,
+    });
+  };
+
+  const parseNewsUrl = async (newsUrl) => {
+    setParsingNews(true);
+    const timeout = setTimeout(coldStartToast, 20000);
+
+    try {
+      const url = `https://z14490usg0.execute-api.us-east-1.amazonaws.com/default/parseNews?url=${encodeURIComponent(
```
Comment (smcgregor) [inline site/gatsby-site/src/components/forms/IncidentReportForm.js:129]: @cesarvarela I am going to merge this PR, but let's move this connection string into the config file. I want to keep all the lambda functions and integrations in one place.

**Pull request merged**